hpc-code

```diff
@@ -0,0 +1,82 @@
+## Running previous code
+
+import pandas as pd
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import mean_absolute_error
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeRegressor
+
+# Set up code checking
+import os
+if not os.path.exists("../input/train.csv"):
+    os.symlink("../input/home-data-for-ml-course/train.csv", "../input/train.csv")  
+    os.symlink("../input/home-data-for-ml-course/test.csv", "../input/test.csv") 
+from learntools.core import binder
+binder.bind(globals())
+from learntools.machine_learning.ex7 import *
+
+# Path of the file to read. We changed the directory structure to simplify submitting to a competition
+iowa_file_path = '../input/train.csv'
+
+home_data = pd.read_csv(iowa_file_path)
+# Create target object and call it y
+y = home_data.SalePrice
+# Create X
+features = ['LotArea', 'YearBuilt', '1stFlrSF', '2ndFlrSF', 'FullBath', 'BedroomAbvGr', 'TotRmsAbvGrd']
+X = home_data[features]
+
+# Split into validation and training data
+train_X, val_X, train_y, val_y = train_test_split(X, y, random_state=1)
```

Comment:
What is the `train_test_split` ratio? I can probably find this from the documentation but I'm just asking you here so you can go through the process of searching the documents. After you find the answer, why do you think it's this specific ratio? What problem does it solve?